6_visualize/src/prep_ocean_name_fun.R

```diff
@@ -0,0 +1,31 @@
+
+prep_ocean_name_fun <- function(ocean_name_placement){
+
+  plot_fun <- function(){
+    # coordinate space (edges, width, height)
+    coord_space <- par()$usr
+    coord_space_bot <- coord_space[3]
+    coord_space_left <- coord_space[1]
+    coord_width <- diff(coord_space[c(1, 2)])
+    coord_height <- diff(coord_space[c(3, 4)])
+
+    ocean_name_w_frac <- 0.1 # fraction of the width of the entire figure
+    ocean_name_w <- ocean_name_w_frac * coord_width # width of ocean name
+
+    # ocean_name_alpha <- 0.4
+
+    # ocean name placement
+    atlantic_x1 <- coord_space_left + coord_width * ocean_name_placement$atlantic$x1
+    atlantic_y1 <- coord_space_bot + coord_height * ocean_name_placement$atlantic$y1
+    g_of_mex_x1 <- coord_space_left + coord_width * ocean_name_placement$g_of_mex$x1
+    g_of_mex_y1 <- coord_space_bot + coord_height * ocean_name_placement$g_of_mex$y1
+
+    x1_coords <- c(atlantic_x1, g_of_mex_x1)
+    y1_coords <- c(atlantic_y1, g_of_mex_y1)
+
+    labs <- c('Atlantic Ocean', 'Gulf of Mexico')
```

Comment:
Have you thought about specifying the label, _and_ the x,y locations in the config? 

It would, for example, be a lot easier that way to just remove atlantic if we zoomed in more (and adjust gulf).

Reply:
yup, can do that easily.

Reply:
![image](https://user-images.githubusercontent.com/4825431/42710881-7c5a1b74-86ab-11e8-8a4d-586f9adb0fec.png)
or if we want to change label: 
![image](https://user-images.githubusercontent.com/4825431/42710952-cbc87188-86ab-11e8-89e2-0b789439b72b.png)

Reply:
:laughing: